index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Abdullah Sakhizada</title>
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;800&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <div class="home-page">
+        <header>
+            <nav id="nav">
+                <a class="logo" href="#"><img src="assets/images/microverse.jfif" alt="my logo" /></a>
+
+                <nav class="hamburger"><img src="assets/images/Union.svg" alt="Menu"></nav>
+                <ul class="internal-link">
+                    <li><a href="">Home</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href=""> Contact </a></li>
+                </ul>
+
+            </nav>
+        </header>
+        <section id="home-header">
+            <h5 class="description">Hello Microverse <br>Abdullah Sakhizada <br>
+                <span class="text-primary">I’m a software developer</span>
+            </h5>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <footer>
+
+                <div id="contact">
+                    <ul>
+                        <li class="icon"><a href="https://www.linkedin.com/in/" rel="noopener noreferrer"
+                                target="_blank"><img src="assets/images/twitter.svg" alt="twitter link"></a></li>
+                        <li class="icon"><a href="https://github.com/" rel="noopener noreferrer" target="_blank"><img
+                                    src="assets/images/Linkedin-icon.svg" alt="linkedin link"></a></li>
+                        <li class="icon"><a href="https://github.com/" target="_blank" rel="noopener noreferrer"><img
+                                    src="assets/images/medium.svg" alt="github link"></a></li>
+                        <li class="icon"><a href="https://github.com/" target="_blank" rel="noopener noreferrer"><img
+                                    src="assets/images/github.svg" alt="github link"></a></li>
+                        <li class="icon"><a href="https://github.com/" rel="noopener noreferrer" target="_blank"><img
+                                    src="assets/images/angellist.svg" alt="github link"></a></li>
+
+
+                    </ul>
+                </div>
+            </footer>
```

Comment:
- [ ] Please, consider not using the `footer` inside of the headline section. You will have to build a `footer` in the next milestones of creating your portfolio page.